lib/node_modules/@stdlib/array/base/cusome-by-right/README.md

```diff
@@ -0,0 +1,147 @@
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# cusomeByRight
+
+> Cumulatively test whether at least `n` array elements in a provided array pass a test implemented by a predicate function, while iterating from right-to-left.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var cusomeByRight = require( '@stdlib/array/base/cusome-by-right' );
+```
+
+#### cusomeByRight( x, n, predicate\[, thisArg ] )
+
+Cumulatively tests whether at least `n` array elements in a provided array pass a test implemented by a `predicate` function, while iterating from right-to-left.
+
+```javascript
+function isPositive( v ) {
+    return v > 0;
+}
+
+var x = [ 1, 1, 0, 0, 0 ];
+
+var y = cusomeByRight( x, 2, isPositive );
+// returns [ false, false, false, false, true ];
+```
+
+#### cusomeByRight.assign( x, n, out, stride, offset, predicate\[, thisArg ] )
+
+Cumulatively tests whether at least `n` array elements in a provided provided array pass a test implemented by a `predicate` function, while iterating from right-to-left and assigns the cumulative values to elements in the output array.
+
+```javascript
+function isPositive( v ) {
+    return v > 0;
+}
+
+var x = [ 1, 1, 0, 0, 0 ];
+var y = [ false, null, false, null, false, null, false, null, false, null ];
+
+var out = cusomeByRight.assign( x, 2, y, 2, 0, isPositive );
+// returns [ false, null, false, null, false, null, false, null, true, null ]
+
+var bool = ( out === y );
+// returns true
+```
+
+The invoked `function` is provided three arguments:
+
+-   `value`: collection element
+-   `index`: collection index
+-   `collection`: input collection
+
+To set the function execution context, provide a `thisArg`.
+
+```javascript
+function isPositive( v ) {
+    this.count += 1;
+    return ( v > 0 );
+}
+
+var x = [ 1, 1, 0, 0, 0 ];
+
+var context = {
+    'count': 0
+};
+
+var bool = cusomeByRight( x, 2, isPositive, context );
+// returns [ false, false, false, false, true ]
+
+var count = context.count;
+// returns 5
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var cusomeByRight = require( '@stdlib/array/base/cusome-by-right' );
+
+function isPositive( value ) {
+    return ( value > 0 );
+}
+
+// Create an array of random values:
+var x = discreteUniform( 10, -10, 10 );
+console.log( x );
+
+// Generate the minimum number of required positive array elements:
+var n = discreteUniform( 1, 1, 10 );
```

Comment:
This is the wrong API. This is generating an array `n`, rather than a scalar `n`.

cc @Planeshifter

Reply:
I updated to 
`var n = 2;`